gk/main.c

```diff
@@ -35,6 +35,14 @@
 #include "gatekeeper_launch.h"
 #include "gatekeeper_lls.h"
 
+#ifdef RTE_MACHINE_CPUFLAG_SSE4_2
+#include <rte_hash_crc.h>
+#define DEFAULT_HASH_FUNC       rte_hash_crc
+#else
+#include <rte_jhash.h>
+#define DEFAULT_HASH_FUNC       rte_jhash
+#endif
+
```

Comment:
The file lls/cache.c also uses this `ifdef` to select a hash function. Maybe it could be moved to a header file, instead of written twice (here and in lls/cache.c)?